src/confluent_kafka/src/Admin.c

```diff
@@ -4219,6 +4311,52 @@ Admin_c_GroupResults_to_py (const rd_kafka_group_result_t **c_result_responses,
         return all_groups_result;
 }
 
+/**
+ *
+ * @brief Convert C ListOffsetsResult response to pyobject.
+ *
+ */
+static PyObject *Admin_c_ListOffsetsResult_to_py (const rd_kafka_ListOffsets_result_t *c_result_event) {
```

Comment:
This function should take `rd_kafka_ListOffsetsResultInfo_t **` as input instead of `rd_kafka_ListOffsets_result_t *`. Conversion from `rd_kafka_ListOffsets_result_t` to `rd_kafka_ListOffsetsResultInfo_t **` should happen in the case block of `Admin_background_event_cb` as per convention that we are following.